appscale/tools/appscale_tools.py

```diff
@@ -482,35 +491,38 @@ def gather_logs(cls, options):
     ]
 
     failures = False
-    for ip in all_ips:
+    for public_ip in all_ips:
       # Get the logs from each node, and store them in our local directory
-      local_dir = os.path.join(location, ip)
-      os.mkdir(local_dir)
-
-      if ip == login_host:
-        os.symlink(local_dir, os.path.join(location, "load-balancer"))
+      local_dir = os.path.join(location, public_ip)
+      utils.mkdir(local_dir)
+
+      # Create symlinks for easier navigation in gathered logs
+      node_info = nodes_dict.get(public_ip)
+      if node_info:
+        private_ip = "{}-private-IP".format(node_info["private_ip"])
+        os.symlink(local_dir, os.path.join(location, private_ip))
+        for role in node_info['jobs']:
+          role_dir = os.path.join(location, "{}-nodes".format(role))
+          utils.mkdir(role_dir)
+          os.symlink(local_dir, os.path.join(role_dir, private_ip))
```

Comment:
These should probably be relative rather than absolute links. Otherwise, if a user creates a tarball and unpacks it elsewhere, the symlinks will be pointing to an absolute location from the original file system.